app/src/main/java/tech/ula/ui/SessionEditFragment.kt

```diff
@@ -199,4 +206,16 @@ class SessionEditFragment : Fragment() {
             else -> 2022
         }
     }
+
+    private fun getListDifferenceAndSetNewFilesystem(prevFilesystems: List<Filesystem>, currentFilesystems: List<Filesystem>) {
+        val uniqueFilesystems = currentFilesystems.subtract(prevFilesystems)
+        if (uniqueFilesystems.isNotEmpty()) {
+            setFilesystemForSessionTo(uniqueFilesystems.first())
+        }
+    }
+
+    private fun setFilesystemForSessionTo(filesystem: Filesystem) {
```

Comment:
I don't love the name of this one. Maybe something like `updateFilesystemDetailsForSession`?